test/jest-puppeteer-vuetify/src/index.js

```diff
@@ -0,0 +1,4 @@
+// Import to register with expect-puppeteer as a side effect
+import './jest-xpaths';
```

Comment:
I'm not sure about how best to organize these. `jest-xpaths` (perhaps ill-named) adds some puppeteer bindings that make XPaths easier. `vuetify-xpaths` generates XPaths for vuetify components. The two clearly synergize and for projects that are set up the way ours is, they will both be needed all the time.

However, both could be useful on their own. `vuetify-xpaths` would be equally useful for use with Selenium, but it wouldn't require `jest-xpaths` at all. `jest-xpaths` would be hypothetically useful for any Puppeteer+Jest tests that wanted to use XPaths cleanly (although that seems less realistic to me).

For the scope of this project, I think this organizational scheme is fine. If we want other projects to start using it though, we will eventually have to consider this. It may be better to split them up now, or we could just cross that bridge when we come to it.

Reply:
Agreed.

In the event we do split them, I think `jest-xpaths` would likely depend on `vuetify-xpaths`.